samples/nrf9160/sb_fota_client/prj.conf

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# General config
+CONFIG_ASSERT=y
+CONFIG_REBOOT=y
+CONFIG_BASE64=y
+CONFIG_NEWLIB_LIBC=y
+CONFIG_FPU=y
+
+# Heap pool
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=2048
+
+# Modem library
+CONFIG_NRF_MODEM_LIB=y
+CONFIG_NRF_MODEM_LIB_SYS_INIT=y
+CONFIG_MODEM_INFO=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_NATIVE=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# Shell
+CONFIG_SHELL=y
+CONFIG_SHELL_WILDCARD=n
```

Comment:
It would maybe be a good idea to have a comment why shell wildcards need to be disabled, because I can't remember the reason anymore. :) There must have been a problem with some command when wildcards are enabled.